Packs/CohesityHelios/Integrations/CohesityHeliosEventCollector/CohesityHeliosEventCollector_test.py

```diff
@@ -0,0 +1,321 @@
+import json
+
+import pytest
+
+from CommonServerPython import arg_to_datetime
+
+
+def test_dedup_elements():
+    from CohesityHeliosEventCollector import adjust_and_dedup_elements, ALERT_TIME_FIELD, AUDIT_LOGS_TIME_FIELD
+    """
+    Case 1:
+    Given a list of 3 elements where all IDs appear in the ID list.
+    We expect the result list to have no elements at all and that the final list length was not changed.
+    """
+    new_elements = [{'id': '1', 'latestTimestampUsecs': 1704096000000000},
+                    {'id': '2', 'latestTimestampUsecs': 1704182400000000},
+                    {'id': '3', 'latestTimestampUsecs': 1704268800000000}]
+    existing_element_ids = ['1', '2', '3']
+    deduped_elements = adjust_and_dedup_elements(new_elements=new_elements, existing_element_ids=existing_element_ids,
+                                                 time_field_name='')
+    assert deduped_elements == []
+    assert len(new_elements) == 3
```

Comment:
Theoretically, dedup functions can be built in a way that the original list is filtered, something I strictly wanted to avoid (I am using its length at the end of the fetch loop to determine if I have more data to pull or not) so I added this test to make sure the length is not changing even if someone makes changes to the code in the future.